platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/LinkedEventIntakeWiring.java

```diff
@@ -32,20 +31,22 @@
  * Wiring for the {@link LinkedEventIntake}.
  *
  * @param eventInput                        the input wire for events to be added to the hashgraph
- * @param pauseInput                        the input wire for pausing the linked event intake
  * @param consensusRoundOutput              the output wire for consensus rounds
  * @param consensusEventsOutput             the output wire for consensus events, transformed from the consensus round
  *                                          output
  * @param keystoneEventSequenceNumberOutput the output wire for the keystone event sequence number
  * @param flushRunnable                     the runnable to flush the intake
+ * @param startSquelchingRunnable           the runnable to start squelching
+ * @param stopSquelchingRunnable            the runnable to stop squelching
  */
 public record LinkedEventIntakeWiring(
         @NonNull InputWire<EventImpl> eventInput,
-        @NonNull InputWire<Boolean> pauseInput,
         @NonNull OutputWire<ConsensusRound> consensusRoundOutput,
         @NonNull OutputWire<List<EventImpl>> consensusEventsOutput,
         @NonNull StandardOutputWire<Long> keystoneEventSequenceNumberOutput,
-        @NonNull Runnable flushRunnable) {
+        @NonNull Runnable flushRunnable,
+        @NonNull Runnable startSquelchingRunnable,
+        @NonNull Runnable stopSquelchingRunnable) {
```

Comment:
Not something to be solved in this PR, but I'm really disliking that we have to bind a bunch of lambdas in a record like this. IMO, when an object becomes complex like this, the right thing to do is to make it into a class. But this is something that probably requires discussion as a larger group.

Reply:
I agree. We should discuss as a group and come up with a plan